converter/fluka/cards/figure_card.py

```diff
@@ -18,27 +18,60 @@ def __str__(self) -> str:
                 line = ""
             else:
                 line = "\n"
-            line += f"{figure.figure_type} {figure.name}"
             if type(figure) is FlukaBox:
-                line += (f" {format_float(figure.x_min, n=16):+#}"
-                         f" {format_float(figure.x_max, n=16):+#}"
-                         f" {format_float(figure.y_min, n=16):+#}"
-                         f" {format_float(figure.y_max, n=16):+#}"
-                         f" {format_float(figure.z_min, n=16):+#}"
-                         f" {format_float(figure.z_max, n=16):+#}")
+                x_min = format_float(figure.x_min, n=16)
+                x_max = format_float(figure.x_max, n=16)
+                y_min = format_float(figure.y_min, n=16)
+                y_max = format_float(figure.y_max, n=16)
+                z_min = format_float(figure.z_min, n=16)
+                z_max = format_float(figure.z_max, n=16)
+                line += (f"* box {figure.name}\n"
+                         f"* X range {x_min:+#}, {x_max:+#}\n"
+                         f"* Y range {y_min:+#}, {y_max:+#}\n"
+                         f"* Z range {z_min:+#}, {z_max:+#}\n"
```

Comment:
Can you add here the same information which user types in the yaptide, namely X,Y,Z size lengths ?

Reply:
ok